checks/raw/shell_download_validate.go

```diff
@@ -609,7 +589,7 @@ func isUnpinnedPipInstall(cmd []string) bool {
 
 	// Any other form of install is unpinned,
 	// e.g. `pip install`.
```

Comment:
these comments can probably be deleted now?

Reply:
IMO they are still useful when understanding the function.